container/Dockerfile

```diff
@@ -74,7 +74,15 @@ RUN apk -U upgrade && \
     tesseract-ocr-data-tha \
     tesseract-ocr-data-tur \
     tesseract-ocr-data-ukr \
-    tesseract-ocr-data-vie
+    tesseract-ocr-data-vie \
+    poppler-data
+
+# XXX waiting on fix upstream bug with poppler utils
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=850163#10
+RUN ln -s /usr/share/poppler/cidToUnicode   /cMap && \
+    ln -s /usr/share/poppler/cidToUnicode   /cidToUnicode && \
+    ln -s /usr/share/poppler/nameToUnicode  /nameToUnicode && \
+    ln -s /usr/share/poppler/unicodeMap     /unicodeMap
```

Comment:
I have a reservation for these lines. I see in [this issue](https://bitbucket.org/merarischroeder/poppler-for-windows/issues/5/missing-language-pack-for-adobe-japan1) that the solution to this problem is to install the `poppler-data` package. Also, I see that we erroneously symlink `/usr/share/poppler/cidToUnicode` to `/cMap`.

So, are these lines actually necessary, or are we covered by installing `poppler-data`?

Reply:
Good catch. You're right. We don't need that. I tested with [this document](https://github.com/deeplow/dangerzone-test-set/blob/main/%25e4%25b8%258d%25e5%2590%258c%25e6%25b5%25b7%25e8%25a1%25a8%25e9%259d%25a2%25e6%25b8%25a9%25e5%25ba%25a6%25e5%25af%25b9%25e5%258d%2597%25e6%25b5%25b7%25e5%258f%25b0%25e9%25a3%258e%25e2%2580%259c%25e6%259d%259c%25e9%25b9%2583%25e2%2580%259d%25e7%259a%2584%25e5%25bd%25b1%25e5%2593%258d%25e8%25af%2595%25e9%25aa%258c.pdf_uncompressed.pdf) and it does not have issues like `Missing language pack for`.

Reply:
Resolved via [6a99bf0](https://github.com/freedomofpress/dangerzone/pull/338/commits/6a99bf0903cf6972455011a12fd6fa4d74ba4cf1).